my_digital_being/static/main.js

```diff
@@ -1454,7 +1454,7 @@ async function testGetActionsForTwitter() {
   }
 }
 
-window.initiateOAuth = initiateOAuth;
+window.initiateOAuth = startOAuthFlow;
```

Comment:
This is typo?

Reply:
I changed it to the startOAuthFlow function since there is no function definition for initiateOAuth, and main.js was throwing an error about unable to initialize initiateOAuth